docs/app/Examples/elements/Segment/Types/SegmentVerticalSegmentExample.js

```diff
@@ -5,9 +5,9 @@ export default class SegmentVerticalSegmentExample extends Component {
   render() {
     return (
       <Segment>
-        <Segment className='vertical'>Te eum doming eirmod, nominati pertinacia argumentum ad his.</Segment>
-        <Segment className='vertical'>Pellentesque habitant morbi tristique senectus.</Segment>
-        <Segment className='vertical'>Eu quo homero blandit intellegebat. Incorrupte consequuntur mei id.</Segment>
+        <Segment aligned='vertical'>Te eum doming eirmod, nominati pertinacia argumentum ad his.</Segment>
+        <Segment aligned='vertical'>Pellentesque habitant morbi tristique senectus.</Segment>
+        <Segment aligned='vertical'>Eu quo homero blandit intellegebat. Incorrupte consequuntur mei id.</Segment>
```

Comment:
As per previous comment, shouldn't `vertical` be implied, by not explicitly specifying `horizontal` ?